src/Query.php

```diff
@@ -91,19 +91,19 @@ function request_parameters() {
             trigger_error('Query#language is deprecated, include it in search_query instead', E_USER_DEPRECATED);
             $search_query .= ' lang:' . $this->language;
         }
-        $search_query .= !empty($this->start_time) ? ' start-date:' . $this->datetime_to_iso8601_utc_string($this->start_time) : '';
-        $search_query .= !empty($this->end_time) ? ' end-date:' . $this->datetime_to_iso8601_utc_string($this->end_time) : '';
+        $search_query .= !empty($this->start_time) ? ' start-date:' . $this->datetime_to_utc_string($this->start_time) : '';
+        $search_query .= !empty($this->end_time) ? ' end-date:' . $this->datetime_to_utc_string($this->end_time) : '';
         return [
             'apikey' => $this->client->api_key,
             'q' => $search_query
         ];
     }
 
-    private function datetime_to_iso8601_utc_string($datetime) {
+    private function datetime_to_utc_string($datetime) {
         if($datetime instanceof \DateTime) {
             $datetime_copy = clone $datetime;
             $datetime_copy->setTimezone(new \DateTimeZone('UTC'));
-            return $datetime_copy->format('Y-m-d\TH:i:s\Z');
+            return $datetime_copy->format('Y-m-d\TH:i:s');
```

Comment:
hmm, I wonder about the "T" here, possibly it's silently dropped by the server side. I will need to take a look at that.

Reply:
Actually discussed in https://github.com/twingly/twingly-search-api-python/pull/38#discussion_r144828545 and it seems like having the T around is good :)